### PR TITLE
Updated CHANGELOG.md.

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+Teleport 6.1 contains multiple new features, improvements, and bug fixes.
+
+## New Features
+
+### U2F for Kubernetes and SSH sessions
+
+Adds an option to authorize with 2nd factor when connecting to a node/k8s cluster.
+
+[RFD 14](https://github.com/gravitational/teleport/blob/master/rfd/0014-session-2FA.md)
+[RFD 15](https://github.com/gravitational/teleport/blob/master/rfd/0015-2fa-management.md)
```
Comment (russjones) [inline CHANGELOG.md:14]: @awly Do you have a short introduction and docs links that would go well in the changelog?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
+
+### Dual Authorization Workflows
+
+Request multiple users to review and approve access requests. Find out more in issue 5007.
+
+[#5071](https://github.com/gravitational/teleport/pull/5071)
```
Comment (russjones) [inline CHANGELOG.md:20]: @fspmarshall  Do you have a short introduction and docs links that would go well in the changelog?

Comment (russjones) [inline CHANGELOG.md:20]: @fspmarshall  Do you have a short introduction and docs links that would go well in the changelog?

File: CHANGELOG.md (hunk 1 above)
Comment (awly) [reply CHANGELOG.md:14]: Didn't get the time to finish https://github.com/gravitational/teleport/pull/6285 yet.
Once it's done, linking to the docs would be easiest.

File: CHANGELOG.md (hunk 2 above)
Comment (fspmarshall) [reply CHANGELOG.md:20]: Unfortunately, the issue itself is still the best source of information right now.

**Review by r0mant — APPROVED** [review]

**Pull request merged**